src/components/ContactCard/ContactIdentity.jsx

```diff
@@ -1,23 +1,18 @@
 import React from 'react'
-import cx from 'classnames'
 import Avatar from 'cozy-ui/transpiled/react/Avatar'
 import withBreakpoints from 'cozy-ui/transpiled/react/helpers/withBreakpoints'
 import { fullContactPropTypes } from '../ContactPropTypes'
 import { getFullContactName, getInitials } from '../../helpers/contacts'
 
 const ContactIdentity = ({ contact, breakpoints: { isMobile } }) => {
   return (
-    <div className="u-flex u-flex-items-center u-flex-column-s u-flex-justify-start-s">
+    <div className="u-flex u-flex-items-center u-flex-column-s u-flex-justify-start-s u-ov-hidden u-maw-100 u-mr-1 u-mr-0-s">
```

Comment:
Nit: This starts to be a bit hard to read. I am not sure that at this point utility classes are preferable to a normal CSS class.